src/main/java/net/minecraftforge/registries/injection/MergeStrategy.java

```diff
@@ -0,0 +1,23 @@
+package net.minecraftforge.registries.injection;
+
+public enum MergeStrategy {
```

Comment:
Please be consistent with how you add curly brackets. It seems you use a new line for each curl bracket in other Forge related classes, but this class appears to be an exception. Just a minor nitpick :P.

Reply:
Ahah thanks good spot, it's fighting my instincts _and_ my IDE to newline opening brackets 😅 I'll do another pass for consistency, think I noticed a try/catch somewhere too that I was meaning to touch up. Nitpicks very much welcome :D